components/wifi/esp8266-driver/mbed_lib.json

```diff
@@ -21,6 +21,14 @@
             "help": "RESET pin for the modem, defaults to Not Connected",
             "value": null
         },
+        "rst-pin-polarity": {
+            "help": "Polarity of RESET pin for the modem. [0/1]",
+            "value": 0
```

Comment:
It might be useful to make use of the "options" field here to require either 0 or 1 as value here.

Reply:
@teetak01 Where can I reference "options" field?

Reply:
["options" example](https://github.com/ARMmbed/mbed-os/blob/b005bf21337e6ffe5393671b92066c3b09c5204b/components/storage/blockdevice/COMPONENT_SPIF/mbed_lib.json#L11)

Reply:
@michalpasztamobica Thanks.
@teetak01 Use `options` for `rst-pin-polarity`.

Reply:
0/1 isn't very meaningful. Have some reference to "active high" and/or "active low". CF `UARTSerial::set_data_carrier_detect`.